### PR TITLE
Regenerated Network SDK

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**

### CODE REVIEW

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.10.1-preview</Version>
+		<Version>20.0.0-preview</Version>
 		<PackageTags>Microsoft Azure Network management;Network;Network management;</PackageTags>    
 		<PackageReleaseNotes>
 		<![CDATA[
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Please update `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj (hunk 1 above)
Comment (EvgenyAgafonchikov) [reply src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: updated

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Comment by MikhailTryakhov** [comment]
@dsgouda  looks like it's ready to merge

**Pull request merged**